scripts/parallel.js

```diff
@@ -0,0 +1,43 @@
+const webdriver = require('selenium-webdriver');
+const { By } = require('selenium-webdriver');
+const assert = require('assert');
+// Input capabilities
+var { parallelTestCapabilities, userName, accessKey } = require('../conf')
+
+async function runTestWithCaps (capabilities) {
+  let driver = new webdriver.Builder().usingServer(`https://${userName}:${accessKey}@hub-cloud.browserstack.com/wd/hub`).withCapabilities(capabilities).build();
+  try{
+    await driver.get("https://bstackdemo.com/");
+    await driver.wait(webdriver.until.titleMatches(/StackDemo/i), 10000);
+
+    // locating product on webpage
+    const productOnScreen = await driver.wait(webdriver.until.elementLocated(By.xpath('//*[@id="1"]/p')), 10000)
+    // getting name of the product when the product is visible
+    const productText =  await driver.wait(webdriver.until.elementIsVisible(productOnScreen, 10000)).getText();
+    // clicking the 'Add to cart' button
+    await driver.wait(webdriver.until.elementIsVisible(driver.findElement(By.xpath('//*[@id="1"]/div[4]'), 10000))).click();
+    // waiting until the Cart pane has been displayed on the webpage
+    await driver.wait(webdriver.until.elementIsVisible(driver.findElement(By.className('float-cart__content'), 10000)));
+    // locating product in cart
+    const productInCart = await driver.wait(webdriver.until.elementLocated(By.xpath('//*[@id="__next"]/div/div/div[2]/div[2]/div[2]/div/div[3]/p[1]')), 10000);
```

Comment:
General comment, as a better practice, can spread long individual lines over multiple lines to make code more readable.
Like here:
```
const productInCart = await driver.wait(
  webdriver.until.elementLocated(
    By.xpath(
      '//*[@id="__next"]/div/div/div[2]/div[2]/div[2]/div/div[3]/p[1]'
    )
  ), 10000
);
```

Reply:
Got it.